fluent-cli/rust-toolchain

```diff
@@ -0,0 +1 @@
+stable
```

Comment:
Why is this in a subdirectory and not the root? What things actually use this? (I would rather it be a separate PR.)

Reply:
Because it's a mistake. Also `fluent-cli` is quite broken in several ways and trying to work around that while still testing other things was making for a tap-dance ... and I tripped.

Reply:
Corrected.